tornjak-frontend/src/components/tornjak-helper.js

```diff
@@ -1,5 +1,8 @@
 // Additional objects the functions take
-// entriesMetaData, agentsSpiffeIdDict, workLoadAttestorInfo, clusterEntry
+// entriesMetaData - https://github.com/spiffe/tornjak/blob/main/docs/tornjak-ui-api-documentation.md#:~:text=GET-,/api/entry/list,-Request%20%0Aapi/entry
```

Comment:
hmm, should this be called just `entries` instead? Entries metadata should be reserved for the additional metadata on the entires that tornjak maintains?

Reply:
Actually this is not even used in the file anymore?

Reply:
removed